### DISCUSSION TITLE
nb进程因为未知原因完全无响应

### DISCUSSION THREAD

**Discussion by justawebsite** [nonebot/discussions#405] [问答]
如图，总是间断性的进入无响应状态，也没有报错，甚至连Ctrl+C都没有反应

只能Ctrl+V强制结束，再杀后台

![image](https://github.com/nonebot/discussions/assets/90131721/9a710ea5-1a0b-442c-877f-ff1228668768)

**Comment by justawebsite** [comment]
这里gocq是没有问题的，十分诡异的是，反向ws显示正常连接

服务器系统是Ubuntu 22.04 Server，nonebot在screen中运行

**Reply by yanyongyu** [reply]
看不出来有什么问题，试试不用screen会不会卡住？

**Reply by justawebsite** [reply]
好的，试一下

**Reply by justawebsite** [reply]
对了，请教您，请问有没有可以详细监测程序状态的软件？目前想确定一下这样卡住的原因

**Reply by yanyongyu** [reply]
守护进程可以监测进程是否退出并重启，如supervisor等。或者docker可以在守护进程的同时，做到对应用的健康进行监测，你可以自己写一个health check。

**Reply by justawebsite** [reply]
谢谢！
